.github/workflows/schema-generator-workflow.yml

```diff
@@ -0,0 +1,40 @@
+name: Build Schema Generator
+
+on:
+  push:
+    branches:
+      - main
+      - 1.*
+    paths:
+      - 'support/checkstyle/**'
+      - 'debezium-schema-generator/**'
+      - 'debezium-parent/pom.xml'
+      - 'debezium-bom/pom.xml'
```

Comment:
@ani-sha the schema generator has a dependency to debezium-core. Can you add this as build change trigger too?
and also for PRs.

Reply:
@rk3rn3r Added the core dependency.